src/main/java/io/stargate/sgv2/jsonapi/api/model/command/NoOptionsCommand.java

```diff
@@ -1,12 +1,29 @@
 package io.stargate.sgv2.jsonapi.api.model.command;
 
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import io.quarkus.runtime.annotations.RegisterForReflection;
+import io.stargate.sgv2.jsonapi.exception.ErrorCode;
+import io.stargate.sgv2.jsonapi.exception.JsonApiException;
 
 /**
  * Interface that Commands that only take "empty" Command (that is, do not expose options but should
  * allow empty JSON Object nonetheless) should implement. <br>
  * NOTE: if {@code Command} starts accepting options, it should NO LONGER implement this interface
- * as combination will not work (options field will not be deserialized).
+ * as combination will probably not work.
  */
-@JsonIgnoreProperties({"options"})
-public interface NoOptionsCommand {}
+@RegisterForReflection
```

Comment:
Without this, Native build CI/ITs failed; adding made them pass. Not 100% sure why but suspecting that default method implementation (annotation included, in particular) would not be retained.
I am  glad we have tests for that set up to catch these problems.